### PR TITLE
Fix generate_instrumentation_bootstrap script on windows 🤞 

### PR DESCRIPTION
**Pull request by owais**
# Description

This should fix the generate scripts on windows.

Also running `eachdist format` instead of `black` to format files post-generation.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Tooling enchancement

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Manually

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [x] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Comment by owais** [comment]
@lzchen could you please test this and confirm if this fixes the issue?

File: scripts/generate_instrumentation_bootstrap.py [hunk 1]
```diff
@@ -91,7 +92,16 @@ def main():
     with open(gen_path, "w") as gen_file:
         gen_file.write(source)
 
-    subprocess.run(["black", "-q", gen_path], check=True)
+    subprocess.run(
```
Comment (lzchen) [inline scripts/generate_instrumentation_bootstrap.py:95]: Curious as to why we are running `black` on this folder?

**Comment by lzchen** [comment]
A success message outputted to the console would be nice :)

File: scripts/generate_instrumentation_bootstrap.py (hunk 1 above)
Comment (owais) [reply scripts/generate_instrumentation_bootstrap.py:95]: Generator just writes out valid, functioning python code without having to worry about formatting. We then use black+isort to make sure generated code is properly formatted.  Forcing the generator to care about formatting world unnecessarily complicated it.

**Review by lzchen — APPROVED** [review]

**Review by ocelotl — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Review by lzchen — APPROVED** [review]

**Pull request merged**